plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -707,7 +708,8 @@ private Optional<SystemTable> getRawSystemTable(ConnectorSession session, Schema
             case ALL_MANIFESTS -> Optional.of(new AllManifestsTable(tableName, table, icebergScanExecutor));
             case MANIFESTS -> Optional.of(new ManifestsTable(tableName, table, getCurrentSnapshotId(table)));
             case FILES -> Optional.of(new FilesTable(tableName, typeManager, table, getCurrentSnapshotId(table), icebergScanExecutor));
-            case ENTRIES -> Optional.of(new EntriesTable(typeManager, tableName, table, icebergScanExecutor));
+            case ALL_ENTRIES -> Optional.of(new EntriesTable(typeManager, tableName, table, MetadataTableType.ALL_ENTRIES, icebergScanExecutor));
+            case ENTRIES -> Optional.of(new EntriesTable(typeManager, tableName, table, MetadataTableType.ENTRIES, icebergScanExecutor));
```

Comment:
nit: static import `MetadataTableType. ENTRIES` and `MetadataTableType.ALL_ENTRIES`